server/config.go

```diff
@@ -56,10 +57,8 @@ type Config struct {
 	// Etcd onlys support seoncds TTL, so here is second too.
 	LeaderLease int64 `toml:"lease" json:"lease"`
 
-	// Log level.
-	LogLevel string `toml:"log-level" json:"log-level"`
```

Comment:
Not only for ansible, some users deploy manually. As far as I know,  we have only one user who deploys manually, so we should help him changing the config before updating.